zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/launcher/SparkInterpreterLauncher.java

```diff
@@ -75,6 +83,30 @@ public SparkInterpreterLauncher(ZeppelinConfiguration zConf, RecoveryStorage rec
         !useProxyUserEnv.equals("false"))) {
       sparkConfBuilder.append(" --proxy-user " + context.getUserName());
     }
+    Path localRepoPath =
+        Paths.get(zConf.getInterpreterLocalRepoPath(), context.getInterpreterSettingId());
+    if (isYarnMode()
+        && getDeployMode().equals("cluster")
+        && Files.exists(localRepoPath)
+        && Files.isDirectory(localRepoPath)) {
+      List<String> extraJars = Lists.newArrayList();
+      try {
+        for (Path jar :
+            Files.newDirectoryStream(
+                localRepoPath,
+                new Filter<Path>() {
+                  @Override
+                  public boolean accept(Path entry) throws IOException {
+                    return Files.isRegularFile(entry);
+                  }
+                })) {
+          extraJars.add(jar.toAbsolutePath().toString());
+        }
+      } catch (IOException e) {
+        LOGGER.error("Cannot make a list of additional jars from localRepo: {}", localRepoPath);
```

Comment:
`LOGGER.error("Cannot make a list of additional jars from localRepo: {}", localRepoPath, e);`?

Reply:
Fixed